ruby/ext/google/protobuf_c/defs.c

```diff
@@ -35,6 +35,11 @@
 // -----------------------------------------------------------------------------
 
 const char* kDescriptorInstanceVar = "descriptor";
+ID descriptor_instancevar_interned;
```

Comment:
Since this is used in many files, seems to make sense to have it in the main file? Then you could just initialize it directly without needing a separate function.